pkg/virt-handler/selinux/labels.go

```diff
@@ -114,8 +133,30 @@ func (se *SELinuxImpl) Restore(dir string) (err error) {
 	return nil
 }
 
+func (*SELinuxImpl) InstallPolicy(dir string) (err error) {
+	for _, policyName := range POLICY_FILES {
+		fileDest := dir + "/" + policyName + ".cil"
+		modules, err := exec.Command("/usr/bin/chroot", "--mount", "/proc/1/ns/mnt", "exec", "--", "/usr/sbin/semodule", "-l").CombinedOutput()
+		if err != nil {
+			return fmt.Errorf("failed to retrive a list of installed modules, err: % v", err)
+		}
+		if !strings.Contains(string(modules), policyName) {
```

Comment:
if the policy changes during an update, do we need to re-install it even though the module appears to already exist?

Reply:
another thought here about updates, I suppose we have to be super careful to not remove permissions from the virt-launcher.cil, otherwise it's unclear to me how that might impact existing VMIs

Reply:
> if the policy changes during an update, do we need to re-install it even though the module appears to already exist?

Yes. Good point. It will need to be re-installed. This code here only protects against random virt-handlers restarts... Not sure how to handle it better.

> another thought here about updates, I suppose we have to be super careful to not remove permissions from the virt-launcher.cil, otherwise it's unclear to me how that might impact existing VMIs

Absolutely, it's very tricky.